src/main/java/com/datalogics/pdf/samples/signature/SignDocument.java

```diff
@@ -223,7 +233,7 @@ private static SignatureAppearanceDisplayItemsSet createSignatureAppearanceDispl
         final SignatureAppearanceDisplayItemsSet displayItems = SignatureAppearanceDisplayItemsSet.newInstance();
 
         // Show nothing
```

Comment:
This is looks good and straight forward. Only thing I'd ask is to remove the comment here since this doesn't hide display items anymore. @datalogics-kam